test/visualizer-layout.test.js

```diff
@@ -338,6 +341,7 @@ test('Visualizer layout - collapse - complex example', function (t) {
   layout.updateScale()
   const actualAfter = [...layout.layoutNodes.values()].map(toValidLink)
   t.deepEqual(actualAfter, ['1 => 2;3', '2 => ', '3 => x3;10', 'x3 => ', '10 => x2', 'x2 => '])
+  t.ok(layout.ejectedLayoutNodeIds.includes('x1'))
```

Comment:
I think it would be useful to additionally assert contents of x's (`.collapsedNodes` ids) to prevent some nasty situations going under the radar in the future

Reply:
i.e. something like this under each `actualAfter` in `visualizer-layout.test.js` (it's important for this file to care about collapse details)
https://github.com/nearform/node-clinic-bubbleprof/blob/404f353fdab3926af661c84156e25a5bdda95aae/test/visualizer-layout-node-allocation.test.js#L364